build.sbt

```diff
@@ -177,13 +178,19 @@ lazy val play25ConductRClientLib = project
   .enablePlugins(CrossPerProjectPlugin)
 
 // Lagom version 1
-lazy val lagom1ConductRBundleLib = project
-  .in(file("lagom1-conductr-bundle-lib"))
+lazy val lagom1ConductRBundleLibJavadsl = project
```

Comment:
Should we name this `lagom1Java...` instead ie being consistent with the lib name and also dropping "DSL" to be consistent with other Java Libs?